templates/common/on-prem/files/keepalived-script-default-ingress.yaml

```diff
@@ -3,4 +3,4 @@ path: "/etc/kubernetes/static-pod-resources/keepalived/scripts/chk_default_ingre
 contents:
   inline: |
     #!/bin/bash
-    /host/bin/oc --kubeconfig /var/lib/kubelet/kubeconfig get ep -n openshift-ingress router-internal-default -o yaml | grep -q 'ip: {{`{{.NonVirtualIP}}`}}$'
+    /host/bin/crictl -r unix:///host/run/crio/crio.sock pods --namespace openshift-ingress --state Ready | grep -q 'router-default-[[:xdigit:]]\{10\}-[[:alnum:]]\{5\}'
```

Comment:
We have recently seen some bugs around older bootimages during scale-up. Since we are going to switch to use crirctl, can you please confirm if  this crictl functionality was present in crictl shipped in OCP 4.1 bootimage?

Reply:
To add to the comment above (because it's not trivial to find this out), OCP 4.1 bootimage corresponds to the RHCOS 410.8.20190520.0

Reply:
![2023-04-26-151604_930x150_scrot](https://user-images.githubusercontent.com/2862927/234586790-0e6b7059-0cc6-4dcf-8e4c-410b40b84195.png)

Seems to me like `crictl` back in OCP 4.1 bootimage contains what we need here, i.e. `pods` command

Reply:
Thanks for confirming!